CMakeLists.txt

```diff
@@ -65,6 +65,9 @@ function(build_ngraph)
     else()
         ngraph_set(NGRAPH_ONNX_IMPORT_ENABLE OFF)
     endif()
+    if(NGRAPH_ONNX_IMPORT_ENABLE)
+        ngraph_set(NGRAPH_ONNX_EDITOR_ENABLE ON)
+    endif()
```

Comment:
Perhaps this should be folded into the block above?
```
    if(NOT (ANDROID OR WINDOWS_STORE OR (MSVC AND (ARM OR AARCH64)) ))
        ngraph_set(NGRAPH_ONNX_IMPORT_ENABLE ON)
        ngraph_set(NGRAPH_ONNX_EDITOR_ENABLE ON)
    else()
        ngraph_set(NGRAPH_ONNX_IMPORT_ENABLE OFF)
        ngraph_set(NGRAPH_ONNX_EDITOR_ENABLE OFF)
    endif()
```

Reply:
As we've discussed offline, the better default state for onnx editor seems to be OFF - removed.
In order to provide CI check, `NGRAPH_ONNX_EDITOR_ENABLE` is set to `ON` in CI config files.